code/processes/filealerter.q

```diff
@@ -41,7 +42,24 @@ find:{[path;match]
 	.lg.o[`alerter;"searching for ",path,"/",match];
 	files:@[system;findstring;()];
 	if[os=`win;files:,/:[path,"\\"; files]]; files};
+
+//-decodes pcap file and sends to tickerplant
+processpcaps:{[path;file]
+	.lg.o[`alerter;"processing pcap file"];
+	.lg.o[`alerter;"decoding pcap file"];
+	table: .pcap.buildtable[hsym `$(path,"/",file)];
+	
+	.lg.o[`alerter;"checking connection to tickerplant"];
+	if[not tickerplanttype in .servers.SERVERS[`proctype]; .servers.startup[]];
+	if[not tickerplanttype in .servers.SERVERS[`proctype]; .lg.e[`alerter;"connection to tickerplant failed"]; :()];
```

Comment:
I think you can get rid of this section
If you have done .servers.startup on initialisation, then it should have registered an interest in the tickerplant
.servers.gethandlebytype which is used by sendtotickerplant will return handle if it has one, or re-open it if it doesn't